packages/azdo-tasks/teamsfx-cli-task/buildandreleasetask/src/utils/commandBuilder.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as tl from 'azure-pipelines-task-lib/task'
+import {ActionInputs, Commands, Strings} from '../constant'
+import {MultipleOptions} from '../enums/multipleOptions'
+import {SingleOptions} from '../enums/singleOptions'
+import {OptionMap} from '../optionMap'
+
+export function BuildCommandString(): string {
+  const commands = tl.getDelimitedInput(ActionInputs.Commands, Strings.NewLine) || []
+  const subCommand = commands.length > 0 ? commands[0] : ''
```

Comment:
Nit: maybe `commands[0] ?? ''` for concise.

Reply:
Agree. I was not sure if commands[0] is undefined when index is out of range, so made it verbose here.